base_rest/http.py

```diff
@@ -2,7 +2,7 @@
 # Copyright 2017 Akretion (http://www.akretion.com).
 # @author Sébastien BEAU <sebastien.beau@akretion.com>
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
-
+# flake8: noqa
```

Comment:
reason for `noqa`?

Reply:
pre-commit forces to "fix" here:
https://github.com/OCA/rest-framework/blob/feb7b9182bc5a4b5569e5f23411b259226276b48/base_rest/http.py#L121
Then if I add:
```
except ValueError as e:
      msg = "Invalid JSON data: %s" % str(e)
      _logger.info("%s: %s", self.httprequest.path, msg)
      raise BadRequest(msg) from e
```
It should be tested on codecov, which seems not efficient for me. I tried actually, but couldn't succeed.
I tried to add `# flake8: W1115` and it didn't work.
If this `noqa` isn't blocking I would leave it as it's.

Reply:
but here you here you are disabling it for the whole file. IMO it's wrong.
IMO is wrong to hide this in the coverage check.
Please roll it back and ignore the coverage color.